cmd/controller-manager/app/config.go

```diff
@@ -35,6 +36,8 @@ type Config struct {
 	Authentication  apiserver.AuthenticationInfo
 	Authorization   apiserver.AuthorizationInfo
 
+	CloudProvider *kubeoptions.CloudProviderOptions
```

Comment:
I think we should start creating the generic set and the set shared between these two binaries. Similar to how we created k8s.io/apiserver and k8s.io/kubernetes/pkg/kubeapiserver.

@sttts @cheftako

Reply:
done, PTAL